docs/architecture/testing/integration.md

```diff
@@ -7,15 +7,11 @@ Integration tests use the built binaries for the CLI and language runtimes. We h
 * [Environment](gh-file:pulumi#sdk/go/common/testing/environment.go#L42): this is lower level than `ProgramTest`. It takes care of creating a temporary directory to run commands in via `Environment.RunCommand`. Use `Environment.ImportDirectory` to import a test scenario into the environment.
 
 ### How and when to use
+
 :::{attention}
-Before running integration tests, make sure to rebuild the binaries your test will use.
-```bash
-# from the repostiory root, build and install `pulumi`
-make build install
-# from sdk/{python,nodejs,go}, build and install the required language runtimes
-cd sdk/python
-make build install
-```
```

Comment:
I think we should leave this info in as an alternative way if you don't want constant rebuilds.

Reply:
Sounds good, re-added it.